fiftyone/utils/transformers.py

```
@@ -1073,6 +1078,317 @@ def __init__(self, config):
+                import inspect
+                sig = inspect.signature(self._model.forward)
+                if 'dataset_index' in sig.parameters:
+                    outputs = self._model(**processed, dataset_index=torch.tensor(0).to(self.device))
+                else:
+                    outputs = self._model(**processed)
+            
+            # Convert model outputs to FiftyOne format
+            self._output_processor._boxes = boxes
+            result = self._output_processor(outputs, image_sizes, self.config.confidence_thresh)
+            
+        except (RuntimeError, ValueError) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)